### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by davorb**

### CODE REVIEW

**Review by jonkerz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,8 +44,8 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/fortytwoish.
+Bug reports and pull requests are welcome on GitHub at https://github.com/RushPlay/fortytwoiiish.
```
Comment (lunich) [inline README.md:47]: Way too many `i`s? :)

File: fortytwoish.gemspec [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
-  spec.license       = "GPL"
+  spec.homepage      = 'https://github.com/rushplay/fortytwoish'
+  spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
-  spec.bindir        = "exe"
+  spec.bindir        = 'exe'
```
Comment (lunich) [inline fortytwoish.gemspec:19]: Shouldn't it be `bin`?

File: lib/fortytwoish/client.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    def configuration
+      @configuration ||= Fortytwoish.configuration
+    end
+
+    def send_message
+      uri = URI 'https://rest.fortytwo.com/1/im'
+      request = Net::HTTP::Post.new(uri.path)
+      request['Content-Type'] = 'application/json; charset=utf-8'
+      request['Authorization'] = "Token #{configuration.token}"
+      request.body = body
```
Comment (lunich) [inline lib/fortytwoish/client.rb]: Let's maybe extract to a separate method? `build_request`? `prepare_request`?

File: spec/fortytwoish/client_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    end
+
+    it { is_expected.to eq '200' }
+  end
+
+  context 'when the server complains' do
+    before do
+      stub_request(:post, 'https://rest.fortytwo.com/1/im').
+        with(body: '{"destinations":[{"number":"46703051158"}],"sms_content":{"message":"hello, world"}}',
+             headers: { 'Authorization': 'Token ', 'Content-Type': 'application/json; charset=utf-8' }).
```
Comment (lunich) [inline spec/fortytwoish/client_spec.rb]: Does it make sense to test it together with configuration here? To have something like `'Token <YOUR-TOKEN-HERE>'`? For positive scenario as well?

File: spec/fortytwoish/client_spec.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        to_return(status: 400)
+
+    end
```
Comment (lunich) [inline spec/fortytwoish/client_spec.rb]: Extra empty line?

File: spec/fortytwoish/client_spec.rb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+    it { is_expected.to eq '400' }
+  end
+
```
Comment (lunich) [inline spec/fortytwoish/client_spec.rb]: Extra empty line?

**Review by lunich — APPROVED** [review]
Looks good. Some really tiny comments.

File: fortytwoish.gemspec (hunk 2 above)
Comment (davorb) [reply fortytwoish.gemspec:19]: This is the way it was autogenerated by `gem`.

**Comment by davorb** [comment]
@lunich addressed in dbbb145d

**Review by lunich — APPROVED** [review]
@davorb good job! 👍

**Pull request merged**